src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/WorkloadRootPath.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#pragma warning disable IDE0240
+#nullable enable
+#pragma warning restore IDE0240
+
+namespace Microsoft.NET.Sdk.WorkloadManifestReader
+{
+    public record WorkloadRootPath(string? Path, bool Installable);
```

Comment:
I added this record as there was a tuple being passed around previously. Made more sense to use an actual defined type.

Reply:
I think this makes it a little harder to review, but it otherwise seems positive